### DISCUSSION TITLE
SCADA unrecongnized mode.

### DISCUSSION THREAD

**Discussion by Nicky0123** [MikaylaFischler/cc-mek-scada#437] [Support] [answered]
When i try to do anything when i use scada command. it just does nothing. says unrecongnized mode.

**Comment by MikaylaFischler** [comment]
Can you give an example of "anything"?

**Reply by Nicky0123** [reply]
now?

**Reply by Nicky0123** [reply]
when i try to use install or other commands

**Reply by MikaylaFischler** [reply, marked as answer]
A valid command with the installer would look like `ccmsi install supervisor`

**Reply by Nicky0123** [reply]
im on 1.19.2 btw

**Reply by MikaylaFischler** [reply]
'scada' isn't a command to clarify

**Reply by Nicky0123** [reply]
i may have named it the wrong way lemme check

**Reply by Nicky0123** [reply]
nah its still not working it says unrecongnized mode with the ccmsi

**Reply by Nicky0123** [reply]
works THANKS!
